### DISCUSSION TITLE
Température multiple

### DISCUSSION THREAD

**Discussion by Qualif22** [community/community#115472] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bonjour,
Est possible de débloqué la limite de entités température (2) car je trouve votre prog superbe et je l’utilise justement pour mes capteurs température de ma maison et non pour une piscine.
Merci

**Comment by Akash1134** [comment]
Hi @Qualif22 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
